rj_gameplay/rj_gameplay/role/unassigned_role.py

```diff
@@ -0,0 +1,17 @@
+import stp
```

Comment:
Sadly, I think importing stp doesn't force the imports of all the subpackages (stp.role, stp.rc, etc.). You have to manually do this to fix the mypy errors below.